spec/facets/ExtractionErrorRunFacet.json

```diff
@@ -0,0 +1,51 @@
+{
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Looking at other JSON spec files use 2-space indents.

Reply:
It requires reformatting anyway I think.

Reply:
Fixed.